docker-compose.yml

```diff
@@ -27,14 +30,14 @@ services:
       - DATABASE_PASSWORD=postgres
       - DATABASE_NAME=block-explorer
       - BLOCKCHAIN_RPC_URL=http://host.docker.internal:3050
+      - BATCHES_PROCESSING_POLLING_INTERVAL=1000
```

Comment:
Decrease batches polling interval to 1 sec, so batches appear much faster on local env